src/ONYXKEYS.ts

```diff
@@ -399,6 +399,8 @@ const ONYXKEYS = {
         EXIT_SURVEY_REASON_FORM_DRAFT: 'exitSurveyReasonFormDraft',
         EXIT_SURVEY_RESPONSE_FORM: 'exitSurveyResponseForm',
         EXIT_SURVEY_RESPONSE_FORM_DRAFT: 'exitSurveyResponseFormDraft',
+        TEST_FORM: 'testForm',
+        TEST_FORM_DRAFT: 'testFormDraft',
```

Comment:
Hmm I'm personally against this approach, we shouldn't "pollute" our Onyx keys with test ones imo. WDYT about using `// @ts-expect-error` to deal with the typing issues instead of creating this test keys? Is it feasible?

Reply:
@VickyStash Another alternative: Just use some of the available Onyx form keys (?)

Reply:
@fabioh8010 Okay, let me try...